app/src/main/java/org/fossasia/openevent/activities/LocationActivtiy.java

```diff
@@ -50,6 +54,17 @@ protected void onCreate(Bundle savedInstanceState) {
         sessionRecyclerView.setLayoutManager(new LinearLayoutManager(this));
         sessionRecyclerView.setAdapter(sessionsListAdapter);
         sessionRecyclerView.setItemAnimator(new DefaultItemAnimator());
+        sessionRecyclerView.addOnItemTouchListener(
+                new RecyclerItemClickListener(getApplicationContext(), new RecyclerItemClickListener.OnItemClickListener() {
+                    @Override
+                    public void onItemClick(View view, int position) {
+                        String session_name = ((TextView) findViewById(R.id.session_title)).getText().toString();
```

Comment:
you should make use of `adapter.getItem(position)` to get the data not through the view.